helm/charts/hydra/templates/job-mirgation.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "hydra.fullname" . }}-automigrate
+  {{- if .Release.Namespace }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+  labels:
+{{ include "hydra.labels" . | indent 4 }}
+    {{- with .Values.deployment.labels }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- with .Values.deployment.annotations }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  template:
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}-automigrate
+        image: {{ .Values.image.repository }}:{{ include "hydra.getImageAlpine" . }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command:
+        - /bin/sh
+        - -c
+        - |
+          set -ex
+          apk add curl 
+          curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+          chmod +x kubectl
+          mv kubectl /usr/local/bin/kubectl
+          [[ -f /etc/migration/migration.json ]] && echo "Migration was done, nothing to do here" && exit 0
+          hydra migrate sql -e --yes --config /etc/config/config.yaml
```

Comment:
The migration job is itself capable of identifying if migrations need to  be applied or not - so no need for workarounds here :)

Reply:
Good to know, will simplify the procedure :) What are your thought on adding `kubectl` to our images? Something like `hydra-debug` which is based on alpine and has some additional packages?

Reply:
👍  for me to have an image with kubectl, we will need it in the future as well. I had to use bitnami already while upgrading kube-prom-stack crds :) And also it should be maintained by us meaning security scans done constantly etc

Reply:
Yeah I guess we could add kubectl to the alpine images? I wouldn't have it in the scratch ones though.